src/components/common/StackButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import styled from '@emotion/styled';
+
+interface StackButtonProps {
+  children: React.ReactNode;
+  url?: string;
+}
+
+const StackButton = ({ children, url }: StackButtonProps) => {
+  return (
+    <Border>
+      <a
+        href={url}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        {children}
+      </a>
+    </Border>
```

Comment:
```suggestion
    <a
      href={url}
      target="_blank"
      rel="noopener noreferrer"
    >
      <Border>{children}</Border>
    </a>
```
이렇게 해야 버튼 클릭할 때 링크 이동 되지 않을까요?

Reply:
[cafefb8](https://github.com/In-N-Out-Dev/PortfolioHub/pull/8/commits/cafefb810582a7f434ff41250d86bfa78f4cbe76) 반영했습니다~